### PR TITLE
Changes to Build, Sign and Publish targets to support building in vsts

### PR DESCRIPTION
**Pull request by vijayrkn**
@mlorbetske @phenning @seancpeters

### CODE REVIEW

File: sign.proj [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+							$(OutDir)src\**\microsoft.templateengine.cli.dll;
+							$(OutDir)src\**\microsoft.templateengine.cli.resources.dll;
+							$(OutDir)src\**\microsoft.templateengine.core.dll;
+							$(OutDir)src\**\microsoft.templateengine.core.contracts.dll;
+							$(OutDir)src\**\microsoft.templateengine.edge.dll;
+							$(OutDir)src\**\microsoft.templateengine.ide.dll;
+							$(OutDir)src\**\microsoft.templateengine.orchestrator.runnableprojects.dll;
+							$(OutDir)src\**\microsoft.templateengine.utils.dll;
+							$(OutDir)test\**\microsoft.templateengine.mocks.dll;
+							$(OutDir)test\**\microsoft.templateengine.testhelper.dll">
```
Comment (seancpeters) [inline sign.proj]: Do we also need to sign the other test assemblies - the ones listed under TestProjects in build.proj?

Comment (vijayrkn) [reply sign.proj]: I saw only these 2 getting packed. Are there other test dlls that need to be shipped?

Comment (seancpeters) [reply sign.proj]: You're correct, the others don't need to be shipped. Makes sense.

**Comment by seancpeters** [comment]
@dotnet-bot test this please

**Comment by seancpeters** [comment]
@dotnet-bot test this please

**Pull request merged**